x-pack/test/observability_functional/apps/observability/pages/alerts/state_synchronization.ts

```diff
@@ -90,18 +106,13 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       expect(timeRange).to.be(expected.timeRange);
     }
 
-    async function assertLogsStreamPageTimeRange(expected: string) {
-      // Only handles relative time ranges
-      const datePickerButton = await testSubjects.find('superDatePickerShowDatesButton');
-      const timerange = await datePickerButton.getVisibleText();
-      expect(timerange).to.be(expected);
-    }
-
     async function setTimeRangeToXDaysAgo(numberOfDays: number) {
       await (await testSubjects.find('superDatePickerToggleQuickMenuButton')).click();
-      const numerOfDaysField = await find.byCssSelector('[aria-label="Time value"]');
-      await numerOfDaysField.clearValueWithKeyboard();
-      await numerOfDaysField.type(numberOfDays.toString());
+      const numberField = await find.byCssSelector('[aria-label="Time value"]');
+      await numberField.clearValueWithKeyboard();
```

Comment:
Was there an issue that led to this change or is it an improvement?

Reply:
Yes, previously the test assumed that "days" was selected as the time unit and it therefore just changed the number. But now the previous date picker state might contain "hours", so the test needs to explicitly set it to "days".